install

```diff
@@ -0,0 +1,28 @@
+-- Install the program by rewriting dofiles, and creating a link to the program
+
+local dest_dir = shell.resolve(select(..., 1) or '_build')
+
+print("Installing to %s...":format(dest_dir))
+
+local copyAndFix(src_file, src_file)
+	local r = fs.open(src_file, 'r')
+	local data = r.readAll()
+	r.close()
+	local w = fs.open(dest, 'w')
+	data = data:gsub('dofile%("', 'dofile("'..dest_dir..'/')
+	w.write(data)
+	w.close()
+end
+
+for _, folder in ipairs({'api', 'programs'})
+	for _, file in pairs(fs.list(folder)) do
+		copyAndFix(folder..'/'..file, dest_dir..'/'..folder..'/'..file)
+	end
+end
+
+for _, prog in pairs(fs.list('programs')) do
+	local f = fs.open('/'..prog, 'w')
+	f.write([[loadfile('%s', getfenv())()]]:format(dest_dir..'/programs/'..prog))
```

Comment:
Turns out `loadfile` takes an env as an argument, so this becomes a simple one-liner